2023-09-21 15:09:27,960:ERROR:error:'NoneType' object has no attribute 'close'
Traceback (most recent call last):
  File "logo_flash_main.py", line 945, in closeEvent
    self.serial.close()
AttributeError: 'NoneType' object has no attribute 'close'
